### DISCUSSION TITLE
Standardjs + babel/eslint-parser ignores eslint rules

### DISCUSSION THREAD

**Discussion by labay11** [babel/babel#13487] [Q&A] [answered]
Hello, I have a project that uses the `standard-js-engine` as linter with the `@babel/eslint-parser` as parser. I would like to configure the parser and include different eslint rules like `max-len` for instance. To do that, I created a `.eslintrc` file but the rules there are ignored, what is the best way to do it?

**package.json**
```
  "scripts": { "lint:js": "standard --verbose | snazzy" },
  "standard": {
    "parser": "@babel/eslint-parser",
    "globals": [
      "__ZESJE_VERSION__",
      "alert",
      "confirm"
      ],
     "ignore": [
      "build",
      "dist",
      "node_modules"
    ]
  }
```

**.babelrc**
```
{
    "presets":[
        "@babel/react",
        "@babel/flow"
    ],
    "plugins": [
        "@babel/syntax-dynamic-import",
        "react-hot-loader/babel",
        "@babel/plugin-proposal-class-properties",
        "@babel/plugin-proposal-object-rest-spread"
    ]
}
```

**.eslintrc**
```
{
  "parser": "@babel/eslint-parser",
  "extends": [
    "standard", "standard-react"
  ],
  "plugins": [
    "import",
    "promise"
  ],
    "rules": {
       "react/jsx-handler-names": "off",
       "react/prop-types": "off",
       "max-len": ["error", { "code": 120 }]
    }
}
```

I am using the latest version of all modules.

**Comment by nicolo-ribaudo** [comment, marked as answer]
I don't know how `standard` works: is it possible that it's just ignoring your ESLint config? That config looks correct.

Otherwise, can you create a small repository I can come to reproduce the problem and understand what's going on?

**Reply by labay11** [reply]
I have created this test project: https://github.com/labay11/test-standard

I also think it is ignoring them, if I run `eslint .` instead of `standard` the rules are applied correctly.

**Reply by nicolo-ribaudo** [reply]
Right, I cloned your project and `standard` completely ignores the config file. I tried adding some syntax errors in `.eslinsrc.json`, but `standard` didn't complain. This has nothing to do with `@babel/eslint-parser`.

It looks like the `standard` CLI ignores `.eslintrc` on purpose (https://github.com/standard/standard/issues/1016), so you have to use `eslint` directly.

**Reply by labay11** [reply]
Thanks! I ended up using `eslint` with the `standard` extension as explained in the `issue`.
